modules/nw-metallb-configure-return-traffic-proc.adoc

```diff
@@ -40,37 +42,47 @@ spec:
   maxUnavailable: 3
   desiredState:
     interfaces:
-      - name: ens4vrf <3>
-        type: vrf <4>
-        state: up
-        vrf:
-          port:
-            - ens4 <5>
-          route-table-id: 2 <6>
-    routes: <7>
+    - name: ens4vrf <3>
+      type: vrf <4>
+      state: up
+      vrf:
+        port:
+        - ens4 <5>
+        route-table-id: 2 <6>
+    - name: ens4 <7>
```

Comment:
@Timdawson264 - I updated the indentation for the `route-table-id: 2` field so hopefully this is good now? I tried creating this in a metal clusterbot instance and it applied successfully.